frontend/dashboard/components/SafeErrorView/SafeErrorView.test.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { userEvent } from '@testing-library/user-event';
+import { SafeErrorView, type SafeErrorViewProps } from './SafeErrorView';
+import { textMock } from '../../../testing/mocks/i18nMock';
+
+const { reload: originalReload } = window.location;
+
+describe('SafeErrorView', () => {
+  beforeEach(() => {
+    Object.defineProperty(window, 'location', {
+      writable: true,
+      value: { reload: jest.fn() },
+    });
+  });
+
+  afterEach(() => {
+    window.location.reload = originalReload;
+  });
+
+  it('should render heading, title and message, with a reload button', () => {
+    renderSafeErrorView({
+      heading: 'All users list',
```

Comment:
It might be a personal opinion, but as you are using the string `All users list` both on line 23 and on line 28, I'm more fan of separating it into a variable and use that both places 😄 
This will also increase maintainability if we for some reason want to change the string - easier to change one place and forget the other. 
I really like how you made the renderSafeErrorView, that's great! 🚀

Reply:
ah, sorry - I did misunderstand your first comment! I agree with this one. 🚀